package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "bundle": "gulp bundle",
     "dev": "gulp preview",
+    "devlive": "export LIVERELOAD=true; gulp preview",
```

Comment:
I used the env var in the past on Ubuntu 20.04 and it didn't work as this is described in the PR. See #115.
I will redo so tests with this PR later.

Reply:
Yes I have seen this, perhaps a specific issue on Ubuntu?